src/components/Banner/Vaccinations100M.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { FacebookShareButton, TwitterShareButton } from 'react-share';
+import { OutlinedButton } from 'components/ButtonSystem';
+import { EventAction, EventCategory, trackEvent } from 'components/Analytics';
+import {
+  BannerContainer,
+  InnerContainer,
+  Body,
+  ButtonsContainer,
+} from './Vaccinations100M.style';
+
+const url = 'https://covidactnow.org';
+const quote = `Over 100M people in the US have received at least one dose of a COVID vaccine! Stay safe and let’s keep it up so we can all get back to normal as soon as possible.`;
+const hashtag = 'COVIDActNow';
+
+const trackShareFacebook = () => {
+  trackEvent(
+    EventCategory.HOMEPAGE_BANNER,
+    EventAction.SHARE,
+    'Facebook: 100M Vaccines',
+  );
+};
+
+const trackShareTwitter = () => {
+  trackEvent(
+    EventCategory.HOMEPAGE_BANNER,
+    EventAction.SHARE,
+    'Twitter: 100M Vaccines',
+  );
+};
+
+const Vaccinations100M: React.FC = () => {
+  return (
+    <BannerContainer>
+      <InnerContainer>
+        <Body>
+          <strong>
+            Over 100M people in the US have received at least one dose of a
+            COVID vaccine!{' '}
+          </strong>
+          <span role="img" aria-label="party popper">
+            🎉
+          </span>{' '}
```

Comment:
Storybook warned me of this, emojis need `role="img"` and `aria-label`

Reply:
Wow, nice. 😁 

I think maybe this broke the spacing after the emoji though.  Do we need to throw in a `&nbsp;` or something?  Not sure...  
![image](https://user-images.githubusercontent.com/206364/113365754-00f6d880-930c-11eb-9ff1-9b7f2bc53bfc.png)